python/lsst/obs/base/_instrument.py

```diff
@@ -162,7 +179,7 @@ def getCamera(self) -> Camera:
         raise NotImplementedError()
 
     @classmethod
-    @lru_cache()
+    # @lru_cache()
```

Comment:
Is this commented line intentional?

Reply:
No. I was debugging a problem where a subclass failed the mypy check unless it also had lru_cache on it.